.ko.yaml

```diff
@@ -1 +1 @@
-defaultBaseImage: registry.k8s.io/build-image/go-runner:v2.3.1-go1.19.4-bullseye.0
+defaultBaseImage: registry.k8s.io/build-image/go-runner:v2.3.1-go1.19.10-bullseye.0
```

Comment:
This base image doesn't exist yet, the e2e tests failed. Please update to a an existing tag.

Reply:
seems this `registry.k8s.io/build-image/go-runner:v2.3.1-go1.19.10-bullseye.0` now exists (tested pull)

Reply:
I don't see it here - https://github.com/kubernetes/k8s.io/blob/main/k8s.gcr.io/images/k8s-staging-build-image/images.yaml

I will just update to use 1.19.7 for now